src/en/guide/conf/config/logging.adoc

```diff
@@ -1,5 +1,11 @@
-Since Grails 3.0, logging is handled by the http://logback.qos.ch[Logback logging framework] and can be configured with the `grails-app/conf/logback.xml` file.
+Logging is handled by http://logback.qos.ch[Logback] and can be configured with the `grails-app/conf/logback-spring.xml` file. See the https://docs.spring.io/spring-boot/how-to/logging.html[Spring Boot Logging] and https://docs.spring.io/spring-boot/reference/features/logging.html#features.logging.logback-extensions[Logging Extensions] for all the available options.
 
-NOTE: Since Grails 5.1.2 support for groovy configuration (`grails-app/conf/logback.groovy`) has been removed (by logback 1.2.9). It is possible to add back groovy configuration by adding the https://github.com/virtualdogbert/logback-groovy-config[logback-groovy-config] library to your project.
+// To be re-included when a Grails 7 release of the plugin gets released
+// See: virtualdogbert/logback-groovy-config#15
+//
+// NOTE: Since Grails 5.1.2 support for groovy configuration (`grails-app/conf/logback.groovy`) has been removed (by logback 1.2.9). It is possible to add back groovy configuration by adding the https://github.com/virtualdogbert/logback-groovy-config[logback-groovy-config] library to your project.
+
+NOTE: The filename `logback.xml` still works but `logback-spring.xml` is now recommended
+
+More information can be found on the official https://logback.qos.ch/documentation.html[Logback documentation].
```

Comment:
Should this be "... in the official ..."?

Reply:
My English sucks I should not be the one writing documentaion :)

Reply:
@gsartori You are doing great! That's why we have reviews 👍